src/network/networkd-nexthop.c

```diff
@@ -300,7 +300,7 @@ static int nexthop_configure(NextHop *nexthop, Link *link) {
 
         link_ref(link);
 
-        r = nexthop_add(link, nexthop, &nexthop);
+        r = nexthop_add(link, nexthop, NULL);
```

Comment:
commit message says "network: constify arguments" but this is removing a parameter? what's the reason for this change?

Reply:
As it is not necessary to update the pointer `nexthop`. The function soon return later without using `nexthop` variable.

Reply:
I see, thanks